### ISSUE TITLE
Fix subpages 404ing

### ISSUE THREAD

**Issue by hackabletype** [dexidp/website#13]
Pages beneath the main page are not loading correctly.

The home page works correctly, but no links going further into the site are loading correctly.

for example: https://dexidp.io/docs/connectors/microsoft/
<img width="1392" alt="Screen Shot 2020-10-05 at 4 37 30 PM" src="https://user-images.githubusercontent.com/4453979/95142707-d3b5dd00-0763-11eb-89fb-b54919ee7a09.png">

**Comment by hackabletype** [comment]
subpages do not appear to build with `make production-build` 
```
make production-build
hugo --minify
Start building sites …

                   | EN
-------------------+-----
  Pages            |  2
  Paginator pages  |  0
  Non-page files   |  0
  Static files     | 19
  Processed images |  0
  Aliases          |  0
  Sitemaps         |  1
  Cleaned          |  0

Total in 231 ms
```

**Comment by hackabletype** [comment]
Looks like the issue is related to the pages "draft" status in the markdown header.

**Comment by sagikazarmark** [comment]
Fixed in #14
